### ISSUE TITLE
Deleted `->list` output

### ISSUE THREAD

**Issue by EcchiNeko** [NoxCaos/amusement-club#115]
When you run `->list` and then accept a direct card sale, the bot post for list gets deleted.

**Comment by NoxCaos** [comment]
Makes sense, because pagination and accept/decline store <userID, messageID> and message ID updates when you call pagination. 
It is not a big deal tbh, but id can be fixed if the bot will store 2 lists instead
